app/routes/groups_.$groupId.tsx

```diff
@@ -3,20 +3,146 @@ import { useParams, useLoaderData, Link } from '@remix-run/react';
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708118009/discord_ncsvl3.png"
+                    alt="Discord Icon"
+                    width={32}
+                    height={32}
+                  />
+                </div>
+                <div className="m-2 rounded-full h-8 w-8 justify-center items-center">
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708118305/twitter_pjiutd.png"
+                    alt="Twitter Icon"
+                    width={32}
+                    height={32}
+                  />
+                </div>
+              </div>
+            </div>
+
+            <div className="flex place-content-evenly flex-wrap">
+              <div className="bg-white mt-2 text-black rounded-3xl px-4 flex flex-row items-center">
+                <div className="m-2 rounded-full h-8 w-8 justify-center items-center">
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708120241/avatar-astronaut_icc950.png"
+                    alt="Astronaut avatar"
+                    width={32}
+                    height={32}
+                  />
+                </div>
+                <p>315 Members</p>
+              </div>
+              <div className="bg-white mt-2 text-black rounded-3xl px-4 flex flex-row items-center">
+                <div className="m-2 rounded-full h-8 w-8 justify-center items-center">
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708121513/location_wf4qsz.png"
+                    alt="GPS Icon"
+                    width={32}
+                    height={32}
+                  />
+                </div>
+                <p>Location, USA</p>
+              </div>
+              <div className="bg-white mt-2 text-black hover:bg-warm hover:text-secondary rounded-3xl px-4 flex flex-row items-center">
+                <div className="m-2 rounded-full h-8 w-8 justify-center items-center">
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708121700/share_hmihwe.png"
+                    alt="Arrowed share icon"
+                    width={32}
+                    height={32}
+                  />
+                </div>
+                <p>Share</p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="min-w-[280px] min-h-[280px] p-2 md:p-10 md:flex md:flex-row">
+          <div className="md:w-1/2 p-2 md:p-4">
+            <h2 className="font-extrabold md:text-2xl">What we are about</h2>
+            <p>{data.group.description}</p>
```

Comment:
Future PR: the text is running together here. I'm not sure of a good way to break out the paragraphs to be a little easier to read.